src/resolve/resolved-resolv-conf.c

```diff
@@ -323,6 +323,25 @@ static int write_stub_resolv_conf_contents(FILE *f, OrderedSet *dns, OrderedSet
         return fflush_and_check(f);
 }
 
+void manager_symlink_stub_to_uplink_resolv_conf(void) {
```

Comment:
actually, we usually prefer if callers ignore return values if they want, not callees.

hence, maybe change this function to return int, and then, when you call it, cast the call to (void) to tell static analyzers that we purposefully ignore the return value.

this also has the benefit that the this function becomes shorter, as you can change `{ log_debug_errno(…); return;}` into ust `return log_debug_errno();`